src/Command/DevEnvCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Frosh\Tools\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DevEnvCommand extends Command
+{
+    public static $defaultName = 'frosh:dev:env';
+    public static $defaultDescription = 'For testshops - add/change robots.txt to stop crawling';
+    private string $envPath;
+
+    public function __construct(string $envPath)
+    {
+        parent::__construct();
+        $this->envPath = $envPath;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
Maybe a console option to remove this again?

Reply:
I have to think a little bit about this, multiple entries can be inside, but it shouldn't be a problem